plugins/module_utils/fusion.py

```diff
@@ -61,7 +61,7 @@
 ENV_APP_ID = "FUSION_APP_ID"  # replaced by ENV_ISSUER_ID
 ENV_HOST = "FUSION_HOST"  # replaced by ENV_API_HOST
 DEP_VER = "2.0.0"
-BASE_PATH = "/api/1.1"
+BASE_PATH = "/api/v1"
```

Comment:
I am not sure if fusion has released this v1 path feature. Maybe lets remain 1.1 ?

Reply:
Right. Changing the PR to draft

Reply:
Now API 1.2 was release.

Reply:
Moved this PR from draft to ready